File: 1526_Minimum_Number_of_Increments_on_Subarrays_to_Form_a_Target_Array/solution.py
# class Solution:
#     def minNumberOperations(self, target: List[int]) -> int:
#         operation, prev = 0, 0
#         for num in target:
#             if num > prev:
#                 operation += num - prev
#             prev = num
#         return operation

class Solution:
    def minNumberOperations(self, target: List[int]) -> int:
        operation = target[0]
        for i in range(len(target) - 1):
            if target[i + 1] > target[i]:
                operation += target[i + 1] - target[i]
        return operation

File: 1526_Minimum_Number_of_Increments_on_Subarrays_to_Form_a_Target_Array/solution.c
int minNumberOperations(int* target, int targetSize) {
    int operation = target[0];
    for (int i = 0; i < targetSize - 1; i++) {
        if (target[i + 1] > target[i]) {
            operation += target[i + 1] - target[i];
        }
    }
    return operation;
}